src/main/java/inf112/app/objects/Robot.java

```diff
@@ -306,6 +314,7 @@ public boolean isDead(){
      */
     public void setCheckPoint(Position p){
         this.checkPoint = p;
+        game.manager.get("assets/Sounds/NewCheckPoint.wav", Sound.class).play(4.0f);
```

Comment:
Her må du legge til en try/catch for NullPointerException. Lydene blir ikke lastet inn for testene.